### ISSUE TITLE
TypeError: Could not connect to API server (Load failed)

### ISSUE THREAD

**Issue by erikpb** [formio/formio#1701]
Getting above error when attempting to login using admin@example.com with password CHANGEME at http://localhost:3001. 
Am I missing something?

**Comment by erikpb** [comment]
When submitting the login form, I'm getting error "Unknown plugin option jsonLogic".

**Comment by brendanbond** [comment]
We just introduced a dependency in the master branch (which we use for development) that are going to require some new documentation changes. If you're running this from source, try passing `NODE_OPTIONS=--no-node-snapshot`

**Comment by erikpb** [comment]
Thanks! Will give it a try.
You say master is used for development, is there a more stable branch? I tried one of the recent tags but getting similar issues.

On 15 Feb 2024, at 04:38, Brendan Bond ***@***.***> wrote:

﻿

We just introduced a dependency in the master branch (which we use for development) that are going to require some new documentation changes. If you're running this from source, try passing NODE_OPTIONS=--no-node-snapshot
